static/js/tippyjs.js

```diff
@@ -155,5 +155,6 @@ export function initialize() {
         interactive: true,
         hideOnClick: true,
         theme: "light-border",
+        placement: "bottom",
```

Comment:
I think the problem here is that the tooltips should have higher z-index than Tippy popovers so that they always appear on top of everything; the bottom placement is I think worse visually than the current side placement for the popover (the mouse kinda occludes it a bit).

Reply:
Since the sidebar is also a popover in small widths, the z-index causes the create stream popover to show below it, if I change the z-index that way. I think we shouldn't go that way since z-index can behave differently in different browsers in these close situations.

Reply:
Can we raise the z-index of the tooltips, rather than lowering that of the popover?

Reply:
That won't help since the z-index of tooltips are limited to that of their parent element. We cannot raise the z-index of the parent element because it will then be stacked above the create stream popover.